Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(grunt) {
       test: {
         options: {
           reporter: 'spec',
-          require: 'coffee-script'
+          require: 'coffee-script/register'
```

Comment:
If we do revert this, we may be locking all future Hubot scripts to < CoffeeScript 1.7. I'm not sure what's the best way to  go about this (perhaps some condition for Travis?), but I personally feel that we should just bump it as it should only affect newly generated scripts.